analytical_engine/java/grape-runtime/src/main/java/com/alibaba/graphscope/parallel/ParallelPropertyMessageManager.java

```diff
@@ -0,0 +1,266 @@
+package com.alibaba.graphscope.parallel;
```

Comment:
Add copyright header please.

Please double check all of other java files under `analytical_engine/java` as well.